frontApp/babel.config.js

```diff
@@ -1,6 +1,7 @@
-module.exports = function(api) {
+module.exports = function (api) {
   api.cache(true);
   return {
     presets: ['babel-preset-expo'],
+    plugins: ['expo-router/babel', 'react-native-reanimated/plugin'],
```

Comment:
does the comma at the end of this line not break the return statement?

Reply:
It doesn't break it but I'll remove it